<launch>
    <arg name="map" default="map_small.yaml" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(find bot_sim)/map/$(arg map)"/>
</launch>
